sdk/python/kfp/dsl/__init__.py

```diff
@@ -43,9 +43,11 @@
 ]
 
 from kfp.components.component_decorator import component
+from kfp.components.container_component_decorator import container_component
```

Comment:
Can you add `container_component` and `ComponentSpec` to `__all__`? This explicitly declares these objects as belonging to the KFP SDK public API and ensures they are documented in our API reference docs.

This is required after the recent merge of master into this branch, but is very easy to miss since it didn't result in any conflicts.